.gitattributes

```diff
@@ -0,0 +1,107 @@
+# From https://github.com/Danimoth/gitattributes/blob/master/Web.gitattributes
+
+# Handle line endings automatically for files detected as text
+# and leave all files detected as binary untouched.
+* text=auto
+
+#
+# The above will handle all files NOT found below
+#
+
+#
+## These files are text and should be normalized (Convert crlf => lf)
+#
+
+# source code
+*.php text
+*.css text
+*.sass text
+*.scss text
+*.less text
+*.styl text
+*.js text eol=lf
```

Comment:
`.ts` and `.tsx` found dead in a ditch

Reply:
LOL i just assumed they had .ts files

Reply:
Technically .ts and .tsx should be covered by `* text=auto`, I just switched between tooflesswolf's PR and this one and no funky LF files got generated

Reply:
Seems like the rest of the text endings are unneeded, but I'm too lazy to figure it out